tests/bsim/bluetooth/host/gatt/notify_stress/nrf5340_cpunet-bt_ll_sw_split.conf

```
@@ -0,0 +1,34 @@
+CONFIG_IPC_SERVICE=y
+CONFIG_MBOX=y
+CONFIG_ISR_STACK_SIZE=1024
+CONFIG_IDLE_STACK_SIZE=256
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_IPC_SERVICE_BACKEND_RPMSG_WQ_STACK_SIZE=512
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```

We switched to using 4096 upstream.

```suggestion
CONFIG_HEAP_MEM_POOL_SIZE=4096
```